prime-router/src/main/kotlin/tokens/Server2ServerAuthentication.kt

```diff
@@ -216,9 +216,12 @@ class FindSenderKeyInSettings(val scope: String, val metadata: Metadata) :
         val workflowEngine = WorkflowEngine.Builder().metadata(metadata).build()
```

Comment:
Should we change the way we use `iss`?  For old keys stored in the Sender, the `iss` would be `orgName.senderName`, but for new keys stored in the organization, the `iss` would be just `orgName`?   Needs discussion on what the purpose of `issuer` is.

OTOH, this will likely require the Sender to change their code.   I'm hoping we can do this refactor without having any customer changes.  But once we've fully moved over, then it would be weird to use `orgName.senderName` as the issue, true?

Reply:
I think that would be weird BUT I think we have code to take a string and get either the org from it or the org and sender if the string has the "." character in it? If we use this code, then the sender can still send us `org.sender` and we can just use our special function to just pull out the org name?

Reply:
One option could be in the follow up work after the keys have been migrated is to change this code to treat issuer first as an organization name and then treat it as a sender if one isn't found.  I think that would keep support for both and allow customers to switch over gracefully.